### ISSUE TITLE
what If image -reel not found?

### ISSUE THREAD

**Issue by nicolasFernandez** [pisi/Reel#179]
If could have the possibility to set some sort of message like '360 view not available'

**Comment by pisi** [comment]
Catching network or server errors on client (particularly for images) is not 100% reliable, however, possible. The real problem is that Reel tries to stay human language agnostic, so such message most likely won't be coded in. But I'm thinking about forwarding the image `error` or `abort` events you'll be able to listened to and act if loading error arrises.

Something is a sense of:

``` js
$('#my_image').reel();
$('#my_image').on('error', function(){
  // This would fire for each image which failed to load
});
```

**Comment by pisi** [comment]
@nicolasFernandez I implemented the above proposal and it works as described therein. Enjoy :)

**Comment by pisi** [comment]
Released today as part of [v1.3.0](/pisi/Reel/tree/v1.3.0)
